letsencrypt/cli.py

```diff
@@ -636,8 +638,12 @@ def preprocess_args(self, args):
                 self.verb = token
                 return reordered
 
-        self.verb = "run"
-        return args + ["run"]
+        if "--standalone" in args and "--installer" not in args and "-i" not in args:
```

Comment:
If you'd like to make another **exception** and rush this PR without tests to get it into PyPI, I'm fine in doing so but I want this functionality removed and the help for `--standalone` updated. If you really feel strongly about it, you can open another PR about the issue later. Not having this functionality is not a showstopper.